assets/dev/js/editor/regions/navigator/elements/commands/base/command-navigator.js

```diff
@@ -0,0 +1,17 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class CommandNavigator extends CommandBase {
+	validateArgs( args ) {
+		this.requireContainer( args );
+
+		const { containers = [ args.container ] } = args;
+
```

Comment:
Check also if the navigator is open.

Reply:
Handled: https://github.com/elementor/elementor/pull/14264/commits/b2be23d45787ac398a1243da5b24230a36c44e0e